### PR TITLE
更新externalsType的三个变量的翻译

### PR DESCRIPTION
**Pull request by a847244052**
更新externalsType的三个变量的翻译

_describe your changes..._

- [x] Read and sign the [CLA][1]. PRs that haven't signed it won't be accepted.
- [x] Make sure your PR complies with the [writer's guide][2].
- [x] Review the diff carefully as sometimes this can reveal issues.
- [x] Do not abandon your Pull Request: [Stale Pull Requests][3].
- **Remove these instructions from your PR as they are for your eyes only.**

[1]: https://github.com/openjs-foundation/EasyCLA#openjs-foundation-cla
[2]: https://webpack.js.org/contribute/writers-guide/
[3]: https://webpack.js.org/contribute/#pull-requests

### CODE REVIEW

**Comment by a847244052** [comment]
@QC-L 
@jacob-lcs

**Review by a847244052 — COMMENTED** [review]
externalsType翻译

File: src/content/configuration/externals.mdx [hunk 1]
```diff
@@ -360,9 +360,9 @@ module.exports = {
 
 ### externalsType.module $#externalstype-module$
 
-Specify the default type of externals as `'module'`. Webpack will generate code like `import * as X from '...'` for externals used in a module.
+将 externals 类型设置为`'module'`， 它告诉 Webpack 在 module 中为外部引用生成形如 `import * as X from '...'`的代码。
```
Comment (jacob-lcs) [inline src/content/configuration/externals.mdx]:
```suggestion
将 externals 类型设置为 `'module'`，webpack 将会在 module 中为外部引用生成形如 `import * as X from '...'` 的代码。
```

File: src/content/configuration/externals.mdx [hunk 2]
```diff
@@ -360,9 +360,9 @@ module.exports = {
...
 
-Make sure to enable [`experiments.outputModule`](/configuration/experiments/#experimentsoutputmodule) first, otherwise webpack will throw errors.
+确保首先把 [`experiments.outputModule`](/configuration/experiments/#experimentsoutputmodule) 设置为`true`, 否则 Webpack将会报错。
```
Comment (jacob-lcs) [inline src/content/configuration/externals.mdx]:
```suggestion
确保首先把 [`experiments.outputModule`](/configuration/experiments/#experimentsoutputmodule) 设置为 `true`, 否则 webpack 将会报错。
```

File: src/content/configuration/externals.mdx [hunk 3]
```diff
@@ -377,7 +377,7 @@ module.exports = {
 
 ### externalsType.node-commonjs
 
-Specify the default type of externals as `'node-commonjs'`. Webpack will import [`createRequire`](https://nodejs.org/api/module.html#module_module_createrequire_filename) from `'module'` to construct a require function for loading externals used in a module.
+将 externals 类型设置为 `'node-commonjs'`， 它告诉 Webpack 在 module 中为外部引用引入`'module'`的 [`createRequire`](https://nodejs.org/api/module.html#module_module_createrequire_filename) 函数构造一个require函数
```
Comment (jacob-lcs) [inline src/content/configuration/externals.mdx]:
```suggestion
将 externals 类型设置为 `'node-commonjs'`，webpack 将从 `module` 中导入 `createRequire` 来构造一个 require 函数，用于加载模块中使用的外部对象。
```

File: src/content/configuration/externals.mdx [hunk 4]
```diff
@@ -390,7 +390,7 @@ module.export = {
 
 ### externalsType.script
```
Comment (jacob-lcs) [inline src/content/configuration/externals.mdx]:
```suggestion
### externalsType.script $#externalstype-script$
```

File: src/content/configuration/externals.mdx [hunk 5]
```diff
@@ -377,7 +377,7 @@ module.exports = {
 
 ### externalsType.node-commonjs
```
Comment (jacob-lcs) [inline src/content/configuration/externals.mdx]:
```suggestion
### externalsType.node-commonjs $#externalstype-node-commonjs$
```

File: src/content/configuration/externals.mdx [hunk 6]
```diff
@@ -390,7 +390,7 @@ module.export = {
 
 ### externalsType.script
 
-Specify the default type of externals as `'script'`. Webpack will Load the external as a script exposing predefined global variables with HTML `<script>` element. The `<script>` tag would be removed after the script has been loaded.
+将 externals 类型设置为 `'script'`， 它告诉 Webpack 通过HTML中的 `<script>` 标签暴露预定义的全局变量的脚本，当脚本被加载完后 `<script>` 标签会被删除。
```
Comment (jacob-lcs) [inline src/content/configuration/externals.mdx]:
```suggestion
将 externals 类型设置为 `'script'`，webpack 将会通过 HTML 中的 `<script>` 标签加载一个脚本，以暴露预定义的全局变量。当脚本被加载完后 `<script>` 标签会被删除。
```

File: src/content/configuration/externals.mdx (hunk 1 above)
Comment (jacob-lcs) [reply src/content/configuration/externals.mdx]: 中英文之间间隔一个空格哈

Comment (jacob-lcs) [reply src/content/configuration/externals.mdx]: 这里的中英文好像不太对应，是参考的 webpack4 的文档吗

Comment (a847244052) [reply src/content/configuration/externals.mdx]: 好的，externalsType应该是v5新特性，v4没找到，谢谢啦

**Review by jacob-lcs — APPROVED** [review]

**Pull request merged**